### PR TITLE
Resource name constraints (5)

### PR DESCRIPTION
**Pull request by tengqm**
xref: #17969, #19099, #18746

### CODE REVIEW

**Comment by tengqm** [comment]
/hold
Pending #19106

File: content/en/docs/concepts/configuration/pod-priority-preemption.md [hunk 1]
```diff
@@ -117,6 +117,9 @@ priority class name to the integer value of the priority. The name is specified
 in the `name` field of the PriorityClass object's metadata. The value is
 specified in the required `value` field. The higher the value, the higher the
 priority.
+The name of a `PriorityClass` object must be a valid
+[DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names),
+and it cannot be prefixed with "`system-`".
```
Comment (sftim) [inline content/en/docs/concepts/configuration/pod-priority-preemption.md]: nit: should this be:
```suggestion
and it cannot be prefixed with `system-`.
```

File: content/en/docs/concepts/configuration/pod-priority-preemption.md [hunk 2]
```diff
@@ -117,6 +117,9 @@ priority class name to the integer value of the priority. The name is specified
 in the `name` field of the PriorityClass object's metadata. The value is
 specified in the required `value` field. The higher the value, the higher the
 priority.
+The name of a `PriorityClass` object must be a valid
```
Comment (sftim) [inline content/en/docs/concepts/configuration/pod-priority-preemption.md]: nit:
```suggestion
The name of a PriorityClass object must be a valid
```

File: content/en/docs/concepts/containers/runtime-class.md [hunk 3]
```diff
@@ -82,6 +82,9 @@ metadata:
 handler: myconfiguration  # The name of the corresponding CRI configuration
 ```
 
+The name of a `RuntimeClass` object must be a valid
```
Comment (sftim) [inline content/en/docs/concepts/containers/runtime-class.md]: nit:
```suggestion
The name of a RuntimeClass object must be a valid
```

File: content/en/docs/concepts/configuration/pod-priority-preemption.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/concepts/configuration/pod-priority-preemption.md]: done

File: content/en/docs/concepts/configuration/pod-priority-preemption.md (hunk 2 above)
Comment (tengqm) [reply content/en/docs/concepts/configuration/pod-priority-preemption.md]: done

File: content/en/docs/concepts/containers/runtime-class.md (hunk 3 above)
Comment (tengqm) [reply content/en/docs/concepts/containers/runtime-class.md]: done

**Pull request merged**